app/models/user.rb

```diff
@@ -2,6 +2,9 @@ class User < ApplicationRecord
   self.ignored_columns = %w[x509_dn_uuid otp_secret_key totp_timestamp]
   include NonNullUuid
 
+  include ::NewRelic::Agent::MethodTracer
```

Comment:
The new relic tracer moves are unfortunate, but I'm not sure we can reference model classes in initializers in the way we were previously.